configure.ac

```diff
@@ -125,13 +125,13 @@ AC_SUBST(PERL)
 # Check for whether to enable MPI and PETSc which will influence our
 # choice of compiler
 #--------------------------------------------------------------------
-COMPILER_CONTROL_ARGS
+ACSM_COMPILER_CONTROL_ARGS
```

Comment:
ACSM == Autoconf Submodule? Should we somehow indicate it's the libmesh autoconf submodule? "LAS_"?

Reply:
I'd been hoping to keep the name more agnostic (while still being [as unique as practical](https://en.wikipedia.org/wiki/ACSM) to avoid namespace conflicts) but I'm not wedded to it.  I just don't want to accidentally imply that this is *only* useful for libMesh.

Reply:
OK, hopefully we don't need to add a second autoconf submodule in libmesh (!!), as then "ACSM" might be ambiguous...

Reply:
For now if we need more shared autoconf files, I think we'll put them in the autoconf-submodule repo, even if (like the existing ax_* files in there) they're copies of upstream, just so we can make local edits easily enough.

Frankly I'd be thrilled if git some day enabled pulling file-by-file from other repos (still qualified by commit hash), though.  If that happens then I'd love to just pull straight from the autoconf archive where possible; IIRC the maintainers there have been pretty responsive if we ever need anything changed upstream, and in the worst case scenario it'd be easy to temporarily replace a file with a local fork *until* changes were merged upstream.